modules/nf-core/fastp/main.nf

```diff
@@ -29,7 +29,8 @@ process FASTP {
     script:
     def single_end = reads.collect().size() == 1
     def args = task.ext.args ?: ''
-    def polyA = trim_polyA ? "--trim_poly_x" : ''
+    // Our team addition to handle metaT
```

Comment:
Maybe add a patch diff file for this too like you have for seqkit?

Reply:
yes.. I was having some issues with nf-core using the schema plugin